crates/matrix-sdk/src/sliding_sync/mod.rs

```diff
@@ -78,6 +80,38 @@ pub struct SlidingSync {
     response_handling_lock: Arc<AsyncMutex<()>>,
 }
 
+#[derive(Debug)]
+pub(super) struct SlidingSyncStickyParameters {
```

Comment:
Sure, since it's very small I've kept it in the same file (it improves readability IMO), but moved it after the `SlidingSyncInner` implementation, just before the tests.